### PR TITLE
fix: update tests for difference on two sets

### LINKED ISSUE #17815: Passes without changing code Perform a Subset Check on Two Sets of Data
#### Describe your problem and - if possible - how to reproduce it
This challenge can be passed without changing the code in the editor at all. Just click "Run the Test" and it passes.

#### Add a Link to the page with the problem
https://learn.freecodecamp.org/coding-interview-prep/data-structures/perform-a-subset-check-on-two-sets-of-data/

#### Tell us about your browser and operating system

* Browser Name: Chrome
* Browser Version: Version 67.0.3396.99 (Official Build) (64-bit)
* Operating System: Ubuntu Linux 16.04

### PR DESCRIPTION
**Pull request by cazyw**
For Data Structures: Perform a Subset Check on Two Sets of Data:
Updates the test assertions as currently the tests pass without any code changes

- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] My article does not contain shortened URLs or affiliate links.

Closes #17815 and is related to https://github.com/freeCodeCamp/freeCodeCamp/pull/18698

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ For example, if <code>setA = ['a','b','c']</code> and <code>setB = ['a','b','d',
 ```yml
 tests:
   - text: Your <code>Set</code> class should have a <code>difference</code> method.
-    testString: assert((function(){var test = new Set(); return (typeof test.difference === 'function')})(), 'Your <code>Set</code> class should have a <code>difference</code> method.');
+    testString: 'assert((function(){var test = new Set(); return (typeof test.difference === "function")})(), "Your <code>Set</code> class should have a <code>difference</code> method.");'
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md]: This change is not necessary, so please revert.

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ For example, if <code>setA = ['a','b','c']</code> and <code>setB = ['a','b','d',
...
   - text: The proper collection was returned
-    testString: assert((function(){var setA = new Set(); var setB = new Set(); setA.add('a'); setA.add('b'); setA.add('c'); setB.add('c'); setB.add('d'); var differenceSetAB = setA.difference(setB); return (differenceSetAB.size() === 2) && DeepEqual(differenceSetAB.values(), [ 'a', 'b' ])})(), 'The proper collection was returned');
+    testString: 'assert((function(){var setA = new Set(); var setB = new Set(); setA.add("a"); setA.add("b"); setA.add("c"); setB.add("c"); setB.add("d"); var differenceSetAB = setA.difference(setB); return (differenceSetAB.size() === 2) && DeepEqual(differenceSetAB.values(), [ "a", "b" ])})(), "The expected collection was not returned");'
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md]: Please change to the following, as the assert messages are no longer used.  Only the `text` is used in this challenge.
```suggestion
    testString: 'assert((function(){var setA = new Set(); var setB = new Set(); setA.add("a"); setA.add("b"); setA.add("c"); setB.add("c"); setB.add("d"); var differenceSetAB = setA.difference(setB); return (differenceSetAB.size() === 2) && DeepEqual(differenceSetAB.values(), [ "a", "b" ])})());'
```

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ For example, if <code>setA = ['a','b','c']</code> and <code>setB = ['a','b','d',
 ```yml
 tests:
   - text: Your <code>Set</code> class should have a <code>difference</code> method.
-    testString: assert((function(){var test = new Set(); return (typeof test.difference === 'function')})(), 'Your <code>Set</code> class should have a <code>difference</code> method.');
+    testString: 'assert((function(){var test = new Set(); return (typeof test.difference === "function")})(), "Your <code>Set</code> class should have a <code>difference</code> method.");'
   - text: The proper collection was returned
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md]:
```suggestion
  - text: Your <code>difference</code> method should return the proper collection.
```

**Review by RandellDawson — DISMISSED** [review]
Please see the requested changes.

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md]: I fixed on behalf of user.

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md]: I fixed on behalf of user.

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md (hunk 3 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-difference-on-two-sets-of-data.english.md]: I fixed on behalf of user.

**Review by Manish-Giri — APPROVED** [review]
LGTM.

**Comment by ojeytonwilliams** [comment]
@cazyw Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :memo:

**Pull request merged**